lib/bonny/controller.ex

```diff
@@ -22,6 +22,7 @@ defmodule Bonny.Controller do
       @version nil
       @scope nil
       @names nil
+      @additionalPrinterColumns nil
```

Comment:
```suggestion
      @additional_printer_columns nil
```

Id like to keep casings elixir-like when they aren't being sent to k8s

Reply:
done